src/components/content-box/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,27 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ContentBox matches snapshot 1`] = `
+.emotion-0 {
+  border: solid rgba(255, 255, 255, 0.1) 2px;
+  overflow: auto;
+  padding: 24px;
+  border-radius: 0.75rem;
+  background: rgba(255, 255, 255, 0.1);
+}
+
+@media (min-width:1280px) {
+  .emotion-0 {
+    padding: 32px;
+  }
+}
+
+<div>
+  <div
+    class="emotion-0"
+  >
+    <div>
+      HELLO WORLD
```

Comment:
Hi Steven, since the "HELLO WORLD" was actually dummy test we setup before for the test file. Please feel free to delete it or make the test file fits your components more.

Reply:
Thank, I perfer address this in #39, since this PR only handle migrate MUI4 to MUI5

Reply:
Sure, then I'll merge this first and let you address in #39.